central/graphql/resolvers/embeddedobjs/node_vuln_context.go

```diff
@@ -0,0 +1,36 @@
+package embeddedobjs
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// nodeVulnContextKey is the key for the *storage.NodeVulnerability value in the context.
+type nodeVulnContextKey struct{}
+
+// nodeVulnContextValue holds the value of the distro in the context.
+type nodeVulnContextValue struct {
+	vuln        *storage.NodeVulnerability
+	lastScanned *types.Timestamp
```

Comment:
It does not look like you are ever setting or getting this field.  Do we need it?

Reply:
Yeah, removed it